bi-connectors/PowerBIConnector/SqlOdbcPBIConnector.pq

```diff
@@ -56,9 +66,29 @@ SqlOdbcPBIConnectorImpl = (Server as text) as table =>
                     UseSSL = 0
                 ],
 
+        // Substract the server from the user input in case if it entered like 'http://localhost' or 'https://srv.com:100500' or 'localhost:0'
```

Comment:
```suggestion
        // Subtract the server from the user input in case it's entered like 'http://localhost' or 'https://srv.com:100500' or 'localhost:0'
```